lib/restforce/db/command.rb

```diff
@@ -56,8 +56,9 @@ def daemonize
       def run(options = {})
         Dir.chdir(Rails.root)
 
-        Restforce::DB::Worker.after_fork
         Restforce::DB.logger = logger
+        Restforce::DB::Worker.after_fork
```

Comment:
This line and the next one read confusingly to me, especially since I didn't initially see `after_fork` in `Restforce::DB::Worker`. I see now that's it coming in from `FileDaemon`, but this might be worth a little comment.